.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
```

Comment:
you took out the release stage, where is it triggered?
do you need to update the path to it?

Reply:
I've separeted the release to a different workflow, in order be triggered manually and not automatically on each commit to master